apps/ledger-live-desktop/tests/specs/services/discover.spec.ts

```diff
@@ -15,16 +16,27 @@ let continueTest = false;
 test.beforeAll(async ({ request }) => {
   // Check that dummy app in tests/utils/dummy-app-build has been started successfully (see playwright.config.ts 'webServer' option for more info)
   try {
-    const response = await request.get("http://localhost:3001");
-    if (response.ok() === true) {
+    const port = await server.start();
+    const response = await request.get(`http://localhost:${port}`);
+    if (response.ok()) {
       continueTest = true;
-      console.info("========> Dummy test app successfully running on port 3001! <=========");
+      console.info(`========> Dummy test app successfully running on port ${port}! <=========`);
+      process.env.MOCK_REMOTE_LIVE_MANIFEST = JSON.stringify(server.manifest(port));
```

Comment:
So this is where the magic happens by passing the manifest when starting the server?